lib/reverse_sentence.rb

```diff
@@ -1,6 +1,75 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n^2) - not so sure
+# Space complexity: O(1) - since I tried to do this in place and didn't create extra array?
 def reverse_sentence(my_sentence)
```

Comment:
This works but your time complexity is O(n) because you never make repeated passes through the string.  You could also improve this by making a helper method to reverse a substring like `def reverse_substring(string, start, end)` you could then call that method to reverse the entire string and each time you wanted to reverse a word.